content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx

```diff
@@ -470,3 +470,20 @@ yarn upgrade prisma@4 @prisma/client@4
 </tab>
 
 </TabbedContent>
+
+## Video guide
+
+For a video walkthrough of the upgrade process and examples of upgrade scenarios, see our livestream on upgrading to Prisma 4:
+
+<div class="videoWrapper">
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/FSjkBrfaoEY"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
```

Comment:
accelerometer & gyroscope are super weird to me 🤔

Reply:
Hm, good point... this was copypasted from elsewhere in docs

Reply:
Looks like it's the default embed code and might be used by YouTube to rotate the video on mobile... Not sure exactly but let's keep it then.

Reply:
Actually it looks like that's part of Youtube's 'Embed' details when you hit 'Share'... maybe it's to enable detecting moving the phone from portrait to landscape? So seems fine.